karspexet/settings.py

```diff
@@ -129,3 +131,18 @@
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
 STATIC_URL = '/static/'
+
+try:
+    with open(BASE_DIR + "/env.json") as env_json:
+        ENV = json.load(env_json)
+except FileNotFoundError:
+    import sys
+    sys.stderr.write("""
```

Comment:
It's nicer if you use the `print` function like so:

```python
import sys
print("Oh, nose.", file=sys.stderr)
```

Also this will print indented, which might look funny, but I think it's fine.